CHANGELOG.md

```diff
@@ -1,5 +1,50 @@
 ## Changelog
 
+### Version v1.16.0-rc.1
+
+- Skip container updates if only CDI is selected
```

Comment:
We definitely need to filter these a bit more strictly.

Reply:
is just a helper script, to filter them better we would need to be more strict with Commit naming so we can use regex filters, or have an LLM action as a helper for this